### ISSUE TITLE
lodash/no-extra-args: false positive with angular.module().constant if function is passed and lodash chain is used in it

### ISSUE THREAD

**Issue by jrencz** [wix/eslint-plugin-lodash#69]
This snippet 

```js
(function () {
  'use strict';

  angular
    .module('whatever', [])
    .constant('foo', foo => _(foo).reduce(bar, []));
})();
```

reports 

```
  4:3  error  Too many arguments passed to `constant` (expected 0)  lodash/no-extra-args
```

While those ones:

```js
(function () {
  'use strict';

  angular
    .module('whatever', [])
    // Array#reduce instead of lodash chained reduce
    .constant('foo', foo => foo.reduce(bar, []));
})();
```

```js
(function () {
  'use strict';

  angular
    .module('whatever', [])
    // _.reduce instead of chain
    .constant('foo', foo => _.reduce(foo, bar, []));
})();
```

```js
(function () {
  'use strict';

  angular
    .module('whatever', [])
     // explicit _.chain().reduce().value() instead of normal chain
    .constant('foo', foo => _
      .chain(foo)
      .reduce(bar, [])
      .value()
    );
})();
```

don't do that.

**Comment by jrencz** [comment]
Thank you!
